### ISSUE TITLE
PO003 - Extract Variables check for content Condition should report the endpoint 

### ISSUE THREAD

**Issue by kurtkanaskie** [apigee/apigeelint#339]
The Condition to check for content does not report the endpoint where the Condition resides.\
It returns the name of the policy instead.\
It does return the correct line and column numbers.\
This example happens to be from the target endpoint.

```
/apiproxy/policies/EV-ExtractBUSerrorCode.xml

║ Line     │ Column   │ Type     │ Message                                                │ Rule ID              ║
╟──────────┼──────────┼──────────┼────────────────────────────────────────────────────────┼──────────────────────╢
║ 26       │ 17       │ error    │ An appropriate check for a message body was not        │ PO003                ║
║          │          │          │ found on the enclosing Step or Flow.                   │                      ║
```

**Comment by kurtkanaskie** [comment]
Similarly for PO001 - policy name is returned but no line numbers.

**Comment by ssvaidyanathan** [comment]
Fixed in v2.31.0
